

public class Main  {

    public static void main(String[] args) throws Exception{
        try {
            UnsafeConstructor uc = new UnsafeConstructor();
        } catch (Exception e) {
            System.out.println("\n" + e.toString() + " catched. Task complete!");
        }
    }
}
